pull_request_template.md

```diff
@@ -1,15 +1,9 @@
 ## Summary ##
+
 **Trello card**: https://trello.com/b/THE_PROJECT_BOARD/THE_CARD
```

Comment:
Since we are changing this, I would change it to: `**Github Issue**: #<Number of issue>`